addons/website/models/ir_http.py

```diff
@@ -322,13 +320,10 @@ def _serve_fallback(cls):
             cls._auth_method_public()
         cls._frontend_pre_dispatch()
         cls._handle_debug()
-        request.params = request.get_http_params()
 
         website_page = cls._serve_page()
         if website_page:
             website_page.flatten()
-            cls._register_website_track(website_page)
-            cls._post_dispatch(website_page)
```

Comment:
restore `_post_dispatch`, website's registery_website_track is now readonly friendly and there are other modules that override post_dispatch and that must be called (e.g. UTM)

Reply:
The call to _post_dispatch is done outside the _serve_fallback